tests/bgp/test_bgp_policy.py

```diff
@@ -0,0 +1,554 @@
+'''
+
+This script is to Verify applied policies manipulate traffic as
+expected for the configured routes.
+
+Configure loopback interfaces to advertise.
+1. Configure and verify policy to permit only Default Route
+2. Configure and verify policy to permit only the first two prefixes
+3. Configure and verify policy with AS-path prepend
+4. Configure and verify AS-path access list using Regexp (matching)
+5. Configure and verify AS-path access list using Regexp (non-matching)
+6. Configure and verify Community-list policy to permit only first two routes
+7. Configure and verify Community-list policy to permit remaining Routes
+8. Configure and verify Community-list policy to permit only first two routes using regexp
+9. Configure and verify Community-list policy to permit remaining Routes using regexp
+
+'''
+import logging
+
+import pytest
+import time
+
+logger = logging.getLogger(__name__)
+dut_4byte_asn = 400003
+neighbor_4byte_asn = 400001
+bgp_sleep = 60
+bgp_id_textfsm = "./bgp/templates/bgp_id.template"
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+
+@pytest.fixture(scope='session')
+def get_test_routes(gather_info):
```

Comment:
this fixture is never used. Why keep it?

Reply:
This has been removed.